packages/components/stories/wizard/wizard.stories.js

```diff
@@ -4,105 +4,93 @@ import { boolean, withKnobs } from '@storybook/addon-knobs';
 import { withInfo } from '@storybook/addon-info';
 import Wizard from 'Components/wizard';
 import Button from 'Components/button';
-import { FlexWrapper, GroupWrapper, ButtonWrapper, Text } from '../button/shared-style';
+import { FlexWrapper, ButtonWrapper, Text } from '../button/shared-style';
 import Theme from '../shared/theme';
 
 const stories = storiesOf('Wizard', module);
 
 stories.addDecorator(withKnobs).addDecorator(withInfo);
 
-stories
-    .add('Basic Usage', () => (
-        <Theme is_dark={boolean('Theme', false)}>
-            <FlexWrapper>
-                <Wizard
-                    steps={[
-                        {
-                            component: <Text size='1.2rem'>Customized component 1</Text>,
-                        },
-                        {
-                            component: <Text size='1.2rem'>Customized component 2</Text>,
-                        },
-                    ]}
-                    has_next
-                    has_prev
-                >
-                    <Wizard.Header>
-                        <Text size='1.6rem'>Customizable Header</Text>
-                    </Wizard.Header>
-                </Wizard>
-            </FlexWrapper>
-        </Theme>
-    ))
-    .add('Labeled Icons', () => (
+const Controller = ({
+    getCurrentStep,
+    getTotalSteps,
+    goToFirstStep,
+    goToStep,
+    goToLastStep,
+    goToNextStep,
+    goToPreviousStep,
+    step,
```

Comment:
`step` prop seems to be redundant. Is it possible to use `getCurrentStep` instead?

Reply:
correct @balakrishna-binary but the point is to show that user can pass extra props to Wizard component and use it there.

Reply:
Alright 👍